config/eigendaflags/cli.go

```diff
@@ -24,7 +24,6 @@ var (
 	SignerPrivateKeyHexFlagName          = withFlagPrefix("signer-private-key-hex")
 	PutBlobEncodingVersionFlagName       = withFlagPrefix("put-blob-encoding-version")
 	DisablePointVerificationModeFlagName = withFlagPrefix("disable-point-verification-mode")
-	WaitForFinalizationFlagName          = withFlagPrefix("wait-for-finalization")
```

Comment:
removing this flag is a breaking change. Can you add note to PR description that we'll need to release a new major version of proxy after we merging this PR ?

Reply:
This flag wasn't in use, so I thought its removal *wouldn't* represent a breaking change. I thought that unused flags were simply ignored?

Reply:
well its not in use but afair its flag actions makes the program crash if its still used.
So image the scenario where someone is on a super old version of proxy and were still using this flag. Now they upgrade to this new version where this flag no longer exists, but we removed the crash at startup. They will never know that this flag is no longer being used (if they use the ENV_VAR instead of flag).